cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -335,24 +339,27 @@ pub enum TypedManifest {
 #[cfg_attr(test, derive(proptest_derive::Arbitrary))]
 #[serde(deny_unknown_fields)]
 pub struct TypedManifestCatalog {
-    pub(super) version: Version<1>,
+    pub version: Version<1>,
     /// The packages to install in the form of a map from install_id
     /// to package descriptor.
     #[serde(default)]
     pub install: ManifestInstall,
     /// Variables that are exported to the shell environment upon activation.
     #[serde(default)]
-    pub(super) vars: ManifestVariables,
+    pub vars: ManifestVariables,
     /// Hooks that are run at various times during the lifecycle of the manifest
     /// in a known shell environment.
     #[serde(default)]
-    pub(super) hook: ManifestHook,
+    pub hook: ManifestHook,
     /// Profile scripts that are run in the user's shell upon activation.
     #[serde(default)]
-    pub(super) profile: ManifestProfile,
+    pub profile: ManifestProfile,
     /// Options that control the behavior of the manifest.
     #[serde(default)]
     pub options: ManifestOptions,
+    /// Service definitions
+    #[serde(default)]
+    pub services: ManifestServices,
```

Comment:
We need a merge from `main` and variation of https://github.com/flox/flox/pull/1708 to prevent empty older Flox versions breaking on updated lockfiles:
```suggestion
    /// Service definitions
    #[serde(default)]
    #[serde(skip_serializing_if = "BTreeMap::is_empty")]
    pub services: ManifestServices,
```

Example from the current branch:
```
(2) ~/projects/flox/flox/cli on HEAD (785d0798) [$]
% just flox init -d ~/tmp
make: Entering directory '/Users/dcarley/projects/flox/flox/pkgdb'
mkdir -p tests;
Generating tests/.gitignore
make: Leaving directory '/Users/dcarley/projects/flox/flox/pkgdb'
~/projects/flox/flox/cli ~/projects/flox/flox
✨ Created environment 'tmp' (aarch64-darwin)

Next:
  $ flox search <package>    <- Search for a package
  $ flox install <package>   <- Install a package into an environment
  $ flox activate            <- Enter the environment
  $ flox edit                <- Add environment variables and shell hooks


(2) ~/projects/flox/flox/cli on HEAD (785d0798) [$] took 8s
% just flox activate -d ~/tmp -- echo    
make: Entering directory '/Users/dcarley/projects/flox/flox/pkgdb'
mkdir -p tests;
Generating tests/.gitignore
make: Leaving directory '/Users/dcarley/projects/flox/flox/pkgdb'
~/projects/flox/flox/cli ~/projects/flox/flox


(2) ~/projects/flox/flox/cli on HEAD (785d0798) [$]
% /run/current-system/sw/bin/flox install -d ~/tmp hello  
❌ ERROR: could not parse lockfile: unknown field `services`, expected one of `version`, `install`, `vars`, `hook`, `profile`, `options`
```

Reply:
Fixed